pkg/model/components/apiserver.go

```diff
@@ -210,13 +211,12 @@ func (b *KubeAPIServerOptionsBuilder) BuildOptions(o interface{}) error {
 		c.AnonymousAuth = fi.Bool(false)
 	}
 
-	// We disable the insecure port from 1.6 onwards
-	if b.IsKubernetesGTE("1.6") {
-		c.InsecurePort = 0
-		glog.V(4).Infof("Enabling apiserver insecure port, for healthchecks (issue #43784)")
-		c.InsecurePort = 8080
-	} else {
-		c.InsecurePort = 8080
+	// We disable the insecure port and address from 1.10 onwards
+	// https://github.com/kubernetes/kubernetes/issues/58951
```

Comment:
The blocker for this is https://github.com/kubernetes/kubernetes/issues/43784 I believe

Reply:
ok, I c.